woocommerce/woocommerce
# Product Block Editor: introduce AttributesComboboxControl component

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





This PR introduces the new `<AttributesComboboxControl />` components to be used in the admin area of the Product Attributes of the application (follow-up).
Its implementation is based on the ComboboxControl core component.

Closes # .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

Test using storybook.

1. Build the Woocommerce plugin

```cli
pnpm --filter='@woocommerce/plugin-woocommerce' build
```

2. Run Storybook

```cli
pnpm --filter='@woocommerce/storybook' storybook
```

4. Go to the AttributesComboboxControl stories
```
http://localhost:6007/?path=/story/product-editor-components-attributescomboboxcontrol--default
```
5. Confirm how the component works

<img width="678" alt="image" src="https://github.com/woocommerce/woocommerce/assets/77539/e13ffdb2-5270-4363-b18a-c762424d9255">

_**Disclaimer: The storybook context differs from the product editor's. Styles, components, etc., behave differently, affecting the UI/UX of it. Anyway, the story is good enough to test part of the implementation.**_ 




### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
